### DISCUSSION TITLE
Why using express instead of Hono?

### DISCUSSION THREAD

**Discussion by phederal** [payloadcms/payload#8149] [Feature Requests & Ideas]
Why using express instead of Hono?
Hono is much faster than Express

**Comment by akhrarovsaid** [comment]
The problem with using the newest, shiniest framework available is that it'll only stay new and shiny temporarily. However development must happen regardless. 

As I write this there are already several newer, shinier alternatives to Hono (For example: ElysiaJS) - why use Hono when you can use Elysia?

Please don't take this personally, but if you are picking your framework purely based on 'speed' then there is something deeply wrong with your process. Nevermind that when you eclipse a certain codebase threshold that ALL frameworks essentially become 'slow,' you pick the right tool for the job. Let's consider ExpressJS.

First and foremost, for a company that expects to be in business for a long time, how easy is it to find Express devs? Easy.
How easy is it to find solutions for problems in ExpressJS? Easy.
How easy is it to find the plugin required to ship your next feature in ExpressJS? Easy.
How easy is it to pickup for a new developer? Easy.

Express has been around for **ten** years now, and will continue to be around for the foreseeable future - built by elite developers with decades of experience, knowledge, and know-how. How many of these new frameworks can say the same?

I have a hard time following and agreeing with the claim that Express isn't performant enough. It most certainly is for 99% of use-cases. 

Express was the right choice. The Payload team are a group of **seriously** talented devs, and it shows with almost every decision they make.

**Comment by arkmech** [comment]
Will Payload CMS ever be designed in a way to be used with any framework instead of just Express?

I think at the very least should be using something like Fastify over Express. I too would like Hono support though lol

**Comment by akhrarovsaid** [comment]
I don't know if the devs have considered swapping frameworks, and I imagine that this task would require quite a large amount of labor. However, Fastify would be an excellent candidate because of the capability to essentially be a [drop-in replacement for express](https://github.com/fastify/fastify-express#readme).

I'd like to reiterate some of my previous points though. What would the benefit be to Payload in switching frameworks? I understand that Fastify and Hono are more 'performant' in terms of reqs/s, but the scale at which you really see those performance improvements is kind of out there. 

The companies that choose Express do so for its' extensibility, stability, and ease of learning. I just can't bring myself to agree, knowing how many more things the Payload devs are trying to ship, to throw in swapping frameworks on top of it all. Aside from reqs/s at huge scale, what would the benefit be?
